lib/_debugger.js

```diff
@@ -1588,7 +1588,8 @@ Interface.prototype.repl = function() {
   this.repl.on('exit', exitDebugRepl);
 
   // Set new
-  this.repl.eval = this.debugEval.bind(this);
+  this.repl.eval = (code, context, filename, callback) => this.debugEval
+                   (code, context, filename, callback);
```

Comment:
Probably better like this, not sure if the linter really picks either up but this would look more like other stuff we use.

```js
this.repl.eval = (code, context, filename, callback) =>
    this.debugEval(code, context, filename, callback);
```